sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Release History
 
-## 1.6.1 (Unreleased)
+## 1.7.0 (Unreleased)
 
 ### Features Added
 
+- Support `listLabels` method to list all the labels in the configuration setting store.
+- Add `tagsFilter` in the option bag for `listConfigurationSettings` method. This feature allows you to filter configuration settings by specifying tags.
```

Comment:
```suggestion
- Add `tagsFilter` in the option bag for `listConfigurationSettings` method. This feature allows filtering configuration settings by specifying tags.
```
?

Reply:
Also show an inline example?

Reply:
I'll show an example inline and link it to the samples